test/integration/failover_stateful_test.lua

```diff
@@ -132,107 +246,103 @@ function g.test_stateboard_restart()
         }})
     end)
 
-    g.stateboard:start()
+    fio.rmtree(g.state_provider.workdir)
```

Comment:
Do you really need it? If you drop state_provider data you don't restart it. You boot completely new state_provider instance.

Reply:
Yes.

1. The intention was to cover the case of clean state provider, as you correctly pointed out.
2. This patch doesn't add anything new. This line originally was above `:stop()`, but sometimes it produces unnecessary warnings in logs, so I moved it here.